### ISSUE TITLE
[Feature request]  the path generated by adding software is not portable!

### ISSUE THREAD

**Issue by fanyangto** [salvadorbs/Asuite#186]
**Is your feature request related to a problem? Please describe.**
When adding a software into the Asuite menu, the path always defaults to $drive than $asuite. This is problematic. Imagine a family home PC used by multiple family members with their own windows account. They will each have their own \portables directory, thus the dirs won't be at the root.  By default, the path added is $drive. This means if I move this dir or make a new copy to a different directory for a family member, the path would no longer be correct. I have to manually modify all of them. 

In other words, the automatically generated s/w path is not portable!

**Describe the solution you'd like**

If you use $asuite as the path, it makes more sense. Assuming asuite is in the same portable dir, the added path should be $asuite/../app1/app1.exe. There should be no manual changes required, no matter where I copy or move the portable folder. 

**Describe alternatives you've considered**

**Additional context**
